### PR TITLE
GNOM-2118 Add Instruction box to Samples tab for experiment edit and new experi…

### PR DESCRIPTION
**Pull request by jdewell**
…ment setup.

### CODE REVIEW

**Comment by jdewell** [comment]
You can use this url to get rid of most of the whitespace issues:
https://github.com/hci-gnomex/gnomex/pull/19/commits/c83f13065549dd30dc7b57f5ed650737065edb47?w=1

File: flex/views/experiment/TabSamplesIllumina.mxml [hunk 1]
```diff
@@ -206,816 +207,824 @@
...
-	
-		
+
+		<mx:VBox verticalGap="5">
+			<mx:TextArea styleName="instructionBox" width="1230" wordWrap="true" id="textarea1" height="120" editable="false" verticalScrollPolicy="off" fontSize="11">
+				<mx:htmlText><![CDATA[1.  Assign a multiplex group number for each sample in the table below. Samples that are to be sequenced in the same lane should be assigned the same multiplex group number.
+2.  Provide a name for each sample.
+3.  Provide the concentration for each sample.
+4.  Type the volume (ul) that will be provided for each sample.
+5.  Specify the number of sequence lanes required for each multiplex group.  (once multiplex groups are selected, make gnomex so that the user can only enter one number per multiplex group and not a number for each sample).
```
Comment (tcourdy) [inline flex/views/experiment/TabSamplesIllumina.mxml]: The stuff in between parentheses are instructions from Brian.  Let's go ahead and remove those from this box.

I will make this a JIRA issue

File: flex/views/experiment/TabSamplesIllumina.mxml [hunk 2]
```diff
@@ -10,194 +10,195 @@
...
-					parentDocument.sampleSetupView.otherSamplePrepMethod.text.replace(/\"/g, "&quot;") : '';				
-				var idOrganism:String	= parentApplication.submitInternalExperiment() && !parentDocument.isEditState()  ? 
+			var otherSamplePrepMethod:String = parentApplication.submitInternalExperiment() && !parentDocument.isEditState() ?
+					parentDocument.sampleSetupView.otherSamplePrepMethod.text.replace(/\"/g, "&quot;") : '';
+			var idOrganism:String = parentApplication.submitInternalExperiment() && !parentDocument.isEditState() ?
 					parentDocument.sampleSetupView.sampleOrganismCombo.selectedItem.@value :
-					(parentDocument.request.@idOrganismSampleDefault != null ? parentDocument.request.@idOrganismSampleDefault : '');;				
-				var otherOrganism:String =  parentApplication.submitInternalExperiment() && !parentDocument.isEditState()  ? 
+					(parentDocument.request.@idOrganismSampleDefault != null ? parentDocument.request.@idOrganismSampleDefault : '');
+			;
```
Comment (tcourdy) [inline flex/views/experiment/TabSamplesIllumina.mxml]: extra semicolon

**Comment by matthewbjorge** [comment]
Looks good

**Comment by tcourdy** [comment]
Looks good.  I will merge and close this.

**Pull request merged**